diff -uNr vlc-qt-1.0.0-orig/config/Dependencies.cmake vlc-qt-1.0.0/config/Dependencies.cmake
--- vlc-qt-1.0.0-orig/config/Dependencies.cmake	2015-12-27 17:07:17.000000000 +0100
+++ vlc-qt-1.0.0/config/Dependencies.cmake	2016-02-06 20:05:12.362864123 +0100
@@ -26,8 +26,6 @@
     MESSAGE("VLC-Qt: Build with Qt4")
 ELSE()
     FIND_PACKAGE(Qt5Core 5.2.0 REQUIRED)
-    FIND_PACKAGE(Qt5Quick 5.2.0 REQUIRED)
-    FIND_PACKAGE(Qt5Widgets 5.2.0 REQUIRED)
 
     MESSAGE("VLC-Qt: Build with Qt5")
 ENDIF()
diff -uNr vlc-qt-1.0.0-orig/config/Package.cmake vlc-qt-1.0.0/config/Package.cmake
--- vlc-qt-1.0.0-orig/config/Package.cmake	2015-12-27 17:07:17.000000000 +0100
+++ vlc-qt-1.0.0/config/Package.cmake	2016-02-06 20:05:56.282184156 +0100
@@ -59,57 +59,3 @@
     DESTINATION ${VlcQtCoreConfigPackageLocation}
     COMPONENT Devel
 )
-
-WRITE_BASIC_PACKAGE_VERSION_FILE(
-    "${CMAKE_BINARY_DIR}/package/VLCQtWidgets/VLCQtWidgetsConfigVersion.cmake"
-    VERSION ${VLCQT_VERSION}
-    COMPATIBILITY AnyNewerVersion
-)
-EXPORT(EXPORT VLCQtWidgetsTargets
-       FILE "${CMAKE_BINARY_DIR}/package/VLCQtWidgets/VLCQtWidgetsTargets.cmake"
-       NAMESPACE VLCQt::
-)
-CONFIGURE_FILE(config/package/VLCQtWidgetsConfig.cmake
-    "${CMAKE_BINARY_DIR}/package/VLCQtWidgets/VLCQtWidgetsConfig.cmake"
-)
-SET(VlcQtWidgetsConfigPackageLocation lib/cmake/VLCQtWidgets)
-INSTALL(EXPORT VLCQtWidgetsTargets
-        FILE VLCQtWidgetsTargets.cmake
-        NAMESPACE VLCQt::
-        DESTINATION ${VlcQtWidgetsConfigPackageLocation}
-)
-INSTALL(
-    FILES
-        config/package/VLCQtWidgetsConfig.cmake
-        "${CMAKE_BINARY_DIR}/package/VLCQtWidgets/VLCQtWidgetsConfigVersion.cmake"
-    DESTINATION ${VlcQtWidgetsConfigPackageLocation}
-    COMPONENT Devel
-)
-
-IF(QT_VERSION MATCHES 5)
-    WRITE_BASIC_PACKAGE_VERSION_FILE(
-        "${CMAKE_BINARY_DIR}/package/VLCQtQml/VLCQtQmlConfigVersion.cmake"
-        VERSION ${VLCQT_VERSION}
-        COMPATIBILITY AnyNewerVersion
-    )
-    EXPORT(EXPORT VLCQtQmlTargets
-           FILE "${CMAKE_BINARY_DIR}/package/VLCQtQml/VLCQtQmlTargets.cmake"
-           NAMESPACE VLCQt::
-    )
-    CONFIGURE_FILE(config/package/VLCQtQmlConfig.cmake
-        "${CMAKE_BINARY_DIR}/package/VLCQtQml/VLCQtQmlConfig.cmake"
-    )
-    SET(VlcQtQmlConfigPackageLocation lib/cmake/VLCQtQml)
-    INSTALL(EXPORT VLCQtQmlTargets
-            FILE VLCQtQmlTargets.cmake
-            NAMESPACE VLCQt::
-            DESTINATION ${VlcQtQmlConfigPackageLocation}
-    )
-    INSTALL(
-        FILES
-            config/package/VLCQtQmlConfig.cmake
-            "${CMAKE_BINARY_DIR}/package/VLCQtQml/VLCQtQmlConfigVersion.cmake"
-        DESTINATION ${VlcQtQmlConfigPackageLocation}
-        COMPONENT Devel
-    )
-ENDIF()
diff -uNr vlc-qt-1.0.0-orig/src/CMakeLists.txt vlc-qt-1.0.0/src/CMakeLists.txt
--- vlc-qt-1.0.0-orig/src/CMakeLists.txt	2015-12-27 17:07:17.000000000 +0100
+++ vlc-qt-1.0.0/src/CMakeLists.txt	2016-02-06 20:04:07.467798510 +0100
@@ -27,8 +27,4 @@
 # Modules #
 ###########
 ADD_SUBDIRECTORY(core)
-ADD_SUBDIRECTORY(widgets)
 
-IF(QT_VERSION MATCHES 5)
-    ADD_SUBDIRECTORY(qml)
-ENDIF()

