diff -uNr a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2016-07-05 12:02:01.582515613 +0200
+++ b/CMakeLists.txt	2016-07-05 12:10:43.911433060 +0200
@@ -36,13 +36,7 @@
 SET(PROJECT_DESCRIPTION "VLC-Qt - Qt bindings for libVLC")
 
 SET(VLCQT_CORE Core)
-SET(VLCQT_WIDGETS Widgets)
-SET(VLCQT_QML Qml)
-SET(VLCQT_PLUGIN_QML PluginQml)
 SET(VLCQT_CORE_NAME ${PROJECT_LIBRARY}Core)
-SET(VLCQT_WIDGETS_NAME ${PROJECT_LIBRARY}Widgets)
-SET(VLCQT_QML_NAME ${PROJECT_LIBRARY}Qml)
-SET(VLCQT_PLUGIN_QML_NAME ${PROJECT_LIBRARY})
 
 INCLUDE(Macros)
 INCLUDE(CompilerConfig)
@@ -54,26 +48,6 @@
 # Compile and set #
 ###################
 ADD_SUBDIRECTORY(src)
-ADD_SUBDIRECTORY(docs)
-
-
-#########
-# Tests #
-#########
-IF(QT_VERSION MATCHES 5 AND NOT MOBILE)
-    ENABLE_TESTING(true)
-    ADD_SUBDIRECTORY(tests)
-
-    IF(NOT MSVC)
-        SET(COVERAGE OFF CACHE BOOL "Coverage report")
-    ENDIF()
-
-    IF(COVERAGE)
-        MESSAGE(STATUS "Generate coverage report using 'coverage' target")
-        INCLUDE(CodeCoverage)
-        SETUP_TARGET_FOR_COVERAGE(coverage ${CMAKE_CTEST_COMMAND} "${CMAKE_BINARY_DIR}/coverage")
-    ENDIF()
-ENDIF()
 
 ##################################
 # CMake Package files generation #
diff -uNr a/config/Dependencies.cmake b/config/Dependencies.cmake
--- a/config/Dependencies.cmake	2016-07-05 12:02:01.582515613 +0200
+++ b/config/Dependencies.cmake	2016-07-05 12:12:48.785854546 +0200
@@ -26,11 +26,6 @@
     MESSAGE(WARNING "Qt4 support is deprecated and will be removed. Please update to Qt 5 soon.")
 ELSE()
     FIND_PACKAGE(Qt5Core 5.2.0 REQUIRED)
-    FIND_PACKAGE(Qt5Quick 5.2.0 REQUIRED)
-    FIND_PACKAGE(Qt5Widgets 5.2.0 REQUIRED)
-
-    FIND_PACKAGE(Qt5QuickTest 5.2.0 REQUIRED)
-    FIND_PACKAGE(Qt5Test 5.2.0 REQUIRED)
 
     MESSAGE(STATUS "Using Qt ${Qt5Core_VERSION}")
 ENDIF()
diff -uNr a/config/Package.cmake b/config/Package.cmake
--- a/config/Package.cmake	2016-07-05 12:02:01.582515613 +0200
+++ b/config/Package.cmake	2016-07-05 12:11:29.330855511 +0200
@@ -60,58 +60,3 @@
     COMPONENT Devel
 )
 
-IF(NOT MOBILE)
-    WRITE_BASIC_PACKAGE_VERSION_FILE(
-        "${CMAKE_BINARY_DIR}/package/VLCQtWidgets/VLCQtWidgetsConfigVersion.cmake"
-        VERSION ${VLCQT_VERSION}
-        COMPATIBILITY AnyNewerVersion
-    )
-    EXPORT(EXPORT VLCQtWidgetsTargets
-           FILE "${CMAKE_BINARY_DIR}/package/VLCQtWidgets/VLCQtWidgetsTargets.cmake"
-           NAMESPACE VLCQt::
-    )
-    CONFIGURE_FILE(config/package/VLCQtWidgetsConfig.cmake
-        "${CMAKE_BINARY_DIR}/package/VLCQtWidgets/VLCQtWidgetsConfig.cmake"
-    )
-    SET(VlcQtWidgetsConfigPackageLocation lib${LIB_SUFFIX}/cmake/VLCQtWidgets)
-    INSTALL(EXPORT VLCQtWidgetsTargets
-            FILE VLCQtWidgetsTargets.cmake
-            NAMESPACE VLCQt::
-            DESTINATION ${VlcQtWidgetsConfigPackageLocation}
-    )
-    INSTALL(
-        FILES
-            config/package/VLCQtWidgetsConfig.cmake
-            "${CMAKE_BINARY_DIR}/package/VLCQtWidgets/VLCQtWidgetsConfigVersion.cmake"
-        DESTINATION ${VlcQtWidgetsConfigPackageLocation}
-        COMPONENT Devel
-    )
-ENDIF()
-
-IF(QT_VERSION MATCHES 5)
-    WRITE_BASIC_PACKAGE_VERSION_FILE(
-        "${CMAKE_BINARY_DIR}/package/VLCQtQml/VLCQtQmlConfigVersion.cmake"
-        VERSION ${VLCQT_VERSION}
-        COMPATIBILITY AnyNewerVersion
-    )
-    EXPORT(EXPORT VLCQtQmlTargets
-           FILE "${CMAKE_BINARY_DIR}/package/VLCQtQml/VLCQtQmlTargets.cmake"
-           NAMESPACE VLCQt::
-    )
-    CONFIGURE_FILE(config/package/VLCQtQmlConfig.cmake
-        "${CMAKE_BINARY_DIR}/package/VLCQtQml/VLCQtQmlConfig.cmake"
-    )
-    SET(VlcQtQmlConfigPackageLocation lib${LIB_SUFFIX}/cmake/VLCQtQml)
-    INSTALL(EXPORT VLCQtQmlTargets
-            FILE VLCQtQmlTargets.cmake
-            NAMESPACE VLCQt::
-            DESTINATION ${VlcQtQmlConfigPackageLocation}
-    )
-    INSTALL(
-        FILES
-            config/package/VLCQtQmlConfig.cmake
-            "${CMAKE_BINARY_DIR}/package/VLCQtQml/VLCQtQmlConfigVersion.cmake"
-        DESTINATION ${VlcQtQmlConfigPackageLocation}
-        COMPONENT Devel
-    )
-ENDIF()
diff -uNr a/src/CMakeLists.txt b/src/CMakeLists.txt
--- a/src/CMakeLists.txt	2016-07-05 12:02:01.590515495 +0200
+++ b/src/CMakeLists.txt	2016-07-05 12:09:49.976124578 +0200
@@ -28,11 +28,3 @@
 ###########
 ADD_SUBDIRECTORY(core)
 
-IF(NOT MOBILE)
-    ADD_SUBDIRECTORY(widgets)
-ENDIF()
-
-IF(QT_VERSION MATCHES 5)
-    ADD_SUBDIRECTORY(qml)
-    ADD_SUBDIRECTORY(plugins/${VLCQT_PLUGIN_QML_NAME})
-ENDIF()

