diff -uNr a/package/Config.in b/package/Config.in
--- a/package/Config.in	2015-12-20 14:11:14.307842349 +0100
+++ b/package/Config.in	2015-12-05 23:05:32.000000000 +0100
@@ -267,6 +267,7 @@
 	source "package/qtuio/Config.in"
 	source "package/quazip/Config.in"
 	source "package/qwt/Config.in"
+	source "package/rpi-car-system/Config.in"
 endif
 	source "package/weston/Config.in"
 	source "package/x11r7/Config.in"
@@ -733,6 +734,8 @@
 	source "package/tremor/Config.in"
 	source "package/vo-aacenc/Config.in"
 	source "package/webrtc-audio-processing/Config.in"
+	source "package/libvlc/Config.in"
+	source "package/vlc-qt/Config.in"
 endmenu
 
 menu "Compression and decompression"

