diff -uNr buildroot-2015.11.1-orig/package/qt5/Config.in buildroot-2015.11.1/package/qt5/Config.in
--- buildroot-2015.11.1-orig/package/qt5/Config.in	2015-12-05 23:05:32.000000000 +0100
+++ buildroot-2015.11.1/package/qt5/Config.in	2015-12-20 14:48:42.455267472 +0100
@@ -45,6 +45,7 @@
 source "package/qt5/qt5websockets/Config.in"
 source "package/qt5/qt5x11extras/Config.in"
 source "package/qt5/qt5xmlpatterns/Config.in"
+source "package/qt5/qt5tools/Config.in"
 comment "legacy compatibility"
 source "package/qt5/qt5quick1/Config.in"
 source "package/qt5/qt5script/Config.in"
diff -uNr buildroot-2015.11.1-orig/package/qt5/qt5tools/Config.in buildroot-2015.11.1/package/qt5/qt5tools/Config.in
--- buildroot-2015.11.1-orig/package/qt5/qt5tools/Config.in	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2015.11.1/package/qt5/qt5tools/Config.in	2015-12-20 14:49:27.122627181 +0100
@@ -0,0 +1,11 @@
+config BR2_PACKAGE_QT5TOOLS
+	bool "qt5tools"
+	select BR2_PACKAGE_QT5BASE
+	help
+	  Qt is a cross-platform application and UI framework for
+	  developers using C++.
+
+	  This package corresponds to qttools and builds QtHelp and QtUiTools 
+	  for the target and Linguist's command line tools for the host.
+
+	  http://qt-project.org
diff -uNr buildroot-2015.11.1-orig/package/qt5/qt5tools/qt5tools.mk buildroot-2015.11.1/package/qt5/qt5tools/qt5tools.mk
--- buildroot-2015.11.1-orig/package/qt5/qt5tools/qt5tools.mk	1970-01-01 01:00:00.000000000 +0100
+++ buildroot-2015.11.1/package/qt5/qt5tools/qt5tools.mk	2015-12-20 14:50:14.401949586 +0100
@@ -0,0 +1,34 @@
+################################################################################
+#
+# qt5tools
+#
+################################################################################
+
+QT5TOOLS_VERSION = $(QT5_VERSION)
+QT5TOOLS_SITE = $(QT5_SITE)
+QT5TOOLS_SOURCE = qttools-opensource-src-$(QT5TOOLS_VERSION).tar.xz
+QT5TOOLS_DEPENDENCIES = qt5base
+QT5TOOLS_INSTALL_STAGING = YES
+
+ifeq ($(BR2_PACKAGE_QT5BASE_LICENSE_APPROVED),y)
+QT5TOOLS_LICENSE = LGPLv2.1 or GPLv3.0
+QT5TOOLS_LICENSE_FILES = LICENSE.GPL LICENSE.LGPL LGPL_EXCEPTION.txt
+else
+QT5TOOLS_LICENSE = Commercial license
+QT5TOOLS_REDISTRIBUTE = NO
+endif
+
+define QT5TOOLS_CONFIGURE_CMDS
+	(cd $(@D); $(HOST_DIR)/usr/bin/qmake)
+endef
+
+define QT5TOOLS_BUILD_CMDS
+	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D)
+endef
+
+define QT5TOOLS_INSTALL_STAGING_CMDS
+	$(TARGET_MAKE_ENV) $(MAKE) -C $(@D) install
+	$(QT5_LA_PRL_FILES_FIXUP)
+endef
+
+$(eval $(generic-package))
